src/test/java/com/opentok/test/OpenTokTest.java

```diff
@@ -1109,52 +1286,38 @@ public void testListStreams()  {
                                 "          \"layoutClassList\" : [] \n" +
                                 "          } ]\n" +
                                 "        }")));
-        try {
```

Comment:
Why are we removing the `try/catch` if this is throwing an exception?

Reply:
https://github.com/opentok/Opentok-Java-SDK/pull/150#discussion_r212040746